src/System.Private.CoreLib/shared/System/Globalization/Calendar.cs

```diff
@@ -605,135 +471,108 @@ private int GetWeekOfYearOfMinSupportedDateTime(int firstDayOfWeek, int minimumD
                 day += 7;
             }
 
-            return (day / 7 + 1);
-        }
-
-        // it would be nice to make this abstract but we can't since that would break previous implementations
-        protected virtual int DaysInYearBeforeMinSupportedYear
-        {
-            get
-            {
-                return 365;
-            }
+            return day / 7 + 1;
         }
 
+        protected virtual int DaysInYearBeforeMinSupportedYear => 365;
 
         // Returns the week of year for the specified DateTime. The returned value is an
         // integer between 1 and 53.
-        //
-
         public virtual int GetWeekOfYear(DateTime time, CalendarWeekRule rule, DayOfWeek firstDayOfWeek)
         {
             if ((int)firstDayOfWeek < 0 || (int)firstDayOfWeek > 6)
             {
                 throw new ArgumentOutOfRangeException(
-                    nameof(firstDayOfWeek), SR.Format(SR.ArgumentOutOfRange_Range,
-                    DayOfWeek.Sunday, DayOfWeek.Saturday));
+                                nameof(firstDayOfWeek),
+                                SR.Format(SR.ArgumentOutOfRange_Range, DayOfWeek.Sunday, DayOfWeek.Saturday));
```

Comment:
Nit: why is this indenting better?  Seems arbitrary now.

Reply:
Fixed